src/components/ValidationTable/helpers.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/prefer-default-export */
+export const getTabsInfo = (validationProcessComponent, pastProcessesComponent) => ([
```

Comment:
May be this could be a variable instead of a function

Reply:
I need it to be a function because I need to send both components ( ValidationProcess & PastProcesses ) with the information I get from useSelector.

![image](https://github.com/eduNEXT/frontend-app-validation-panel/assets/86393372/3266b304-5aa1-4c1b-8b2c-e4a817555e83)